thrift/thrift0.13/src/main/resources/com/linecorp/armeria/internal/common/thrift/thrift-options.properties

```
@@ -0,0 +1 @@
+structPreinitRequired=true
```

- IMO, `/src/main/resources/armeria-thrift-options.properties` would suffice for the file path and name like:
https://github.com/line/armeria/blob/c3bd51e4655e45a7d7c70f3e4a2cf43adfa88af6/testing-internal/src/main/resources/junit-platform.properties#L1
- `Required` looks redundant.
```suggestion
struct.preinit=true
```

---

> IMO, /src/main/resources/armeria-thrift-options.properties would suffice for the file path and name like:

I don't really have a preference between about the directory location.
If we expect `armeria-thrift-options.properties` to be configurable by users, then I think including in the top-level directory makes more sense. My understanding was that this isn't part of the public API though 😅  (let me know if I'm misunderstanding)

Currently, I prefer as-is just because I can type less `../`, but let me know if you feel otherwise

> Required looks redundant.

Done 👍